project/MiMa.scala

```diff
@@ -11,57 +11,54 @@
  * Copyright (C) 2009-2020 Lightbend Inc. <https://www.lightbend.com>
  */
 
+import scala.collection.immutable
 import sbt._
 import sbt.Keys._
-import com.typesafe.tools.mima.core.ProblemFilter
 import com.typesafe.tools.mima.plugin.MimaPlugin
 import com.typesafe.tools.mima.plugin.MimaPlugin.autoImport._
 
-import scala.util.Try
-
 object MiMa extends AutoPlugin {
 
+  private val latestPatchOf10 = 0
```

Comment:
Split this change to a seprate PR?

Reply:
One advantage to having it here is this PR proves it works. I could move it in a separate PR but the build will pass regardless.

Reply:
I would prefer moving this to a separate PR as well, its not directly related to this change and its also something I would like to review in isolation (its also scratching this pet peeve of mine that I have echoed elsewhere which is that that we shouldn't even have this code, i.e. it should be automated via https://github.com/sbt/sbt-dynver?tab=readme-ov-file#previous-version-detection.